### PR TITLE
Deadline banner

### PR DESCRIPTION
**Pull request by TomBr0wn**
Added banner component to alert the user to closing of claim window
app/views/claims/list.njk
and
app/views/support/organisations/claims/list.njk

https://github.com/DFE-Digital/itt-mentoring-beta-prototype/assets/78021112/6b6a1bcd-d41e-4af7-bd30-841f800d7c59

### CODE REVIEW

File: app/views/claims/list.njk [hunk 1]
```diff
@@ -13,6 +13,11 @@
   {% include "_includes/notification-banner.njk" %}
 
   <div class="govuk-grid-row">
+
+    {% block deadlineBanner %}
+        {% include "_includes/deadline-banner.njk" %}
+    {% endblock %}
+
     <div class="govuk-grid-column-full">
```
Comment (simonwhatley) [inline app/views/claims/list.njk:21]: The banner should sit within a grid row and a full column.
```suggestion
  <div class="govuk-grid-row">
    <div class="govuk-grid-column-full">
      {% include "_includes/deadline-banner.njk" %}
    </div>
  </div>

  <div class="govuk-grid-row">
    <div class="govuk-grid-column-full">
```

File: app/views/start.njk [hunk 2]
```diff
@@ -8,6 +8,11 @@
 
 {% block content %}
   <div class="govuk-grid-row">
+
+    {% block deadlineBanner %}
+        {% include "_includes/deadline-banner.njk" %}
+    {% endblock %}
+
     <div class="govuk-grid-column-two-thirds">
```
Comment (simonwhatley) [inline app/views/start.njk:16]:
```suggestion
  <div class="govuk-grid-row">
    <div class="govuk-grid-column-full">
      {% include "_includes/deadline-banner.njk" %}
    </div>
  </div>

  <div class="govuk-grid-row">
    <div class="govuk-grid-column-full">
```

File: app/views/support/organisations/claims/list.njk [hunk 3]
```diff
@@ -17,6 +17,11 @@
   {% include "_includes/notification-banner.njk" %}
 
   <div class="govuk-grid-row">
+
+      {% block deadlineBanner %}
+        {% include "_includes/deadline-banner.njk" %}
+      {% endblock %}
+
     <div class="govuk-grid-column-two-thirds">
```
Comment (simonwhatley) [inline app/views/support/organisations/claims/list.njk:25]: I'm not sure you need the claim banner in the support site as they typically won't be creating claims.

```suggestion
  <div class="govuk-grid-row">
    <div class="govuk-grid-column-full">
      {% include "_includes/deadline-banner.njk" %}
    </div>
  </div>

  <div class="govuk-grid-row">
    <div class="govuk-grid-column-full">
```

File: app/views/_includes/deadline-banner.njk [hunk 4]
```diff
@@ -0,0 +1,12 @@
+{% set deadlineBanner %}
+
+      <p class="govuk-notification-banner__heading">
+        Final closing date for claims: <strong>19 July 2024 by 11.59 pm</strong>.
+      </p>
+
+
+{% endset %}
```
Comment (simonwhatley) [inline app/views/_includes/deadline-banner.njk:8]:
```suggestion
{% set deadlineBanner %}
  <p class="govuk-notification-banner__heading">
    Final closing date for claims: <strong>19 July 2024 by 11.59 pm</strong>
  </p>
{% endset %}
```

**Review by simonwhatley — CHANGES_REQUESTED** [review]

File: app/views/start.njk [hunk 5]
```diff
@@ -8,7 +8,13 @@
...
 {% block content %}
   <div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
+    <div class="govuk-grid-column-full">
+      {% include "_includes/deadline-banner.njk" %}
+    </div>
+  </div>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
```
Comment (simonwhatley) [inline app/views/start.njk:17]: The page heading should only ever span two-thirds width

```suggestion
    <div class="govuk-grid-column-two-thirds">
```

File: app/views/support/organisations/claims/list.njk [hunk 6]
```diff
@@ -17,7 +17,13 @@
...
 
   <div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
+    <div class="govuk-grid-column-full">
+      {# {% include "_includes/deadline-banner.njk" %} #}
+    </div>
+  </div>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
```
Comment (simonwhatley) [inline app/views/support/organisations/claims/list.njk:26]: The page heading should only ever span two-thirds width

```suggestion
    <div class="govuk-grid-column-two-thirds">
```

**Review by simonwhatley — CHANGES_REQUESTED** [review]

File: app/views/support/organisations/claims/list.njk [hunk 7]
```diff
@@ -17,7 +17,13 @@
   {% include "_includes/notification-banner.njk" %}
 
   <div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
+    <div class="govuk-grid-column-full">
+      {# {% include "_includes/deadline-banner.njk" %} #}
```
Comment (simonwhatley) [inline app/views/support/organisations/claims/list.njk:21]: If you're hiding the banner in support, it's probably a good idea to hide the whole grid row now that it is empty.

**Review by simonwhatley — APPROVED** [review]

File: app/views/start.njk [hunk 8]
```diff
@@ -7,6 +7,12 @@
 {% endblock %}
 
 {% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full-width">
```
Comment (simonwhatley) [inline app/views/start.njk:11]: The class name is `govuk-grid-column-full`:
```suggestion
    <div class="govuk-grid-column-full">
```

**Review by simonwhatley — CHANGES_REQUESTED** [review]

**Comment by TomBr0wn** [comment]
@simonwhatley can you review again please

**Comment by simonwhatley** [comment]
[Introduce a banner into the service to remind users of the deadline and encourage them to apply for funding](https://trello.com/c/b873PnPj/542-introduce-a-banner-into-the-service-to-remind-users-of-the-deadline-and-encourage-them-to-apply-for-funding)

**Pull request merged**